addons/mrp_subcontracting/models/stock_picking.py

```diff
@@ -28,13 +28,9 @@ def _compute_display_action_record_components(self):
             if all(subcontracted_move.has_tracking == 'none' for subcontracted_move in component_sub_moves):
```

Comment:
IMO, you can remove also this condition? It is equivalent to the _has_components_to_record check , no ?